.github/workflows/make-branch.yaml

```diff
@@ -24,5 +25,8 @@ jobs:
         run: |
           git config --global user.name "Mykhailo Kuznietsov"
           git config --global user.email "mkuznets@redhat.com"
-          export GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }}
-          .github/bin/make-branch.sh --branch ${{ github.event.inputs.branch }} --branchfrom ${{ github.event.inputs.branchfrom }} ${{ github.event.inputs.forceflag }}
+          curl https://raw.githubusercontent.com/eclipse-che/che-release/main/make-branch.sh | bash -s -- \
```

Comment:
could add a `-sSL` for quieter console log and to follow symlinks but not sure if required. It's just what I do for all my curls.